app/build.gradle.kts

```diff
@@ -48,7 +48,7 @@ dependencies {
     implementation(Deps.constraintlayout)
     implementation(Deps.SqlDelight.runtimeJdk)
     implementation(Deps.SqlDelight.driverAndroid)
-    implementation(Deps.Coroutines.jdk)
+    implementation(Deps.Coroutines.common)
```

Comment:
This doesn't need to change (it's still supported to use platform-specific versions in JVM- or JS-specific modules) but it also shouldn't hurt.